Description: ubuntu template: set -e to return error on failures.
 Otherwise callers can get bad containers without knowing it.
 This will be forwarded upstream
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/922645
Forwarded: yes

Index: lxc-0.8.0~rc1/templates/lxc-ubuntu.in
===================================================================
--- lxc-0.8.0~rc1.orig/templates/lxc-ubuntu.in	2012-04-23 23:09:19.579940778 -0500
+++ lxc-0.8.0~rc1/templates/lxc-ubuntu.in	2012-04-23 23:09:28.627940625 -0500
@@ -24,6 +24,8 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+set -e
+
 if [ -r /etc/default/lxc ]; then
     . /etc/default/lxc
 fi
